README.adoc

```diff
@@ -44,14 +44,19 @@ toc::[]
 |No
 
 |https://tour.golang.org/[golang]
-a|A Tour of Go, the Google scripting language. If you want to run the tutorial locally, first install https://golang.org/doc/install[Go] then run:
```

Comment:
This may've removed the formatting for L48-51

Reply:
Ah, gotcha. I have put it back in.